docs/release-notes-ps-8.4.md

```diff
@@ -0,0 +1,69 @@
+
+LTS releases are more suitable for stable, established environments where minimal changes are needed. These releases include only essential fixes, reducing the risk of changes in the database software’s behavior.
+
+This 8.4.0-1 release is the first 8.4 LTS series.
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.4 and included in Percona Server for MySQL are the following:
+
+* The MySQL native password has been deprecated and is no longer loaded by default. However, it can be loaded if needed.
+
+* The clone plugin allows cloning between different point releases within the same series. You only must match the major and minor version numbers for cloning.
+
+* GTIDs (Global Transaction Identifiers) can now handle groups of transactions, which helps speed up processing.
+
+* `mysqldump` can now create output for older versions of MySQL.
+
+* Automatic updates for histograms. When enabled, the histogram updates automatically whenever `ANALYZE TABLE` is run on the table. InnoDB's automatic recalculation of persistent statistics also updates the histogram when automatic updates are enabled.
+
+* Adds a new privilege called FLUSH_PRIVILEGES. This privilege explicitly allows the use of FLUSH PRIVILEGES statements. Unlike the RELOAD privilege, FLUSH_PRIVILEGES only applies to FLUSH PRIVILEGES statements.
+
+* The terms "MASTER" and "SLAVE" in replication commands are being replaced with "SOURCE" and "REPLICA". This change is part of an ongoing effort to use more inclusive language.
+
+* Removed the the `mysqlpump` utility.
+
+* Removed the `mysql_upgrade` utility.
+
+* The default values for specific InnoDB server system variables have changed. See [What is new in MySQL 8.4 since 8.0](https://dev.mysql.com/doc/refman/8.4/en/mysql-nutshell.html) for details.
+
+## Deprecation
+
+* [PS-8963](https://perconadev.atlassian.net/browse/PS-8963): The `SEQUENCE_TABLE()` function is deprecated and may be removed in a future release. We recommend that you use `PERCONA_SEQUENCE_TABLE()` instead. To maintain compatibility with existing third-party software, `SEQUENCE_TABLE` is no longer a reserved term and can be used as a regular identifier. Find more information in [PERCONA_SEQUENCE_TABLE(n) function](https://docs.percona.com/percona-server/8.4/percona-sequence-table.html).
+
+## Packaging notes
+
+Percona Server for MySQL 8.4.0-1 is compatible with Ubuntu 24.04.
+
+## Supplied components
+
+Review each component’s release notes for What’s new, improvements, or bug fixes. The following is a list of the components supplied with the Percona Server for MySQL-based variation of the Percona Distribution for MySQL:
+
+| Component           | Version   | Description                                |
+| ------------------- | --------- | -------------------------------------------|
+| Orchestrator        | [3.2.6-13](https://github.com/percona/orchestrator/releases/tag/v3.2.6-13)     | The replication topology manager for Percona Server for MySQL|
```

Comment:
Please double check the exact version of the Orchestrator package.   Latest version is `3.2.6-14` (https://github.com/percona/orchestrator/tags). This is the one that will go into PDPS 8.4.  Please double check the exact package before merging. Thank you!

Reply:
Thank you! Fixed.